core/app/resources/schedule_resource.rb

```diff
@@ -2,11 +2,27 @@ class ScheduleResource < ApplicationResource
   self.adapter = Graphiti::Adapters::Null
 
   def base_scope
-    { }
+    { sort: {} }
+  end
+
+  sort_all do |scope, attribute|
```

Comment:
i think you should include the direction here and store the attribute and direction. you can then use the direction to conditionally call `reverse` in the `resolve` block